docs/Course Wiki/ECE Course Offerings/ECE313.md

```diff
@@ -0,0 +1,83 @@
+
+## When to Take it
+
+It is heavily advised to take this course after [ECE210](ECE210.md), since this course does use convolution. ECE313 will reach convolution before ECE210, hence taking these concurrently will not help either. While ECE313 does fully introduce convolution, the introduction is extremely rushed.
+
+Many computer engineers choose to delay this course until their final semester, as it does not open up many upper-level computer engineering courses. While this may seem like a wise decision in order to avoid a negative impact on your GPA, be warned that the likelihood of failing this class is not zero, especially during your final semester when you are mentally checked out.
+
+Electrical engineers are advised to take this course the semester after they take ECE210, as many upper-level electrical engineering courses in areas such as signal processing, communications, controls, and remote sensing require ECE313 as a prerequisite (and for a good reason). This course is also extremely important for those interested in theoretical computer science or machine learning.
+
+## Course Structure
+
+During the semester, the course assigns 12-13 written homeworks, due every week, and 12 PrairieLearn homeworks (with infinite tries), due every week. There are two midterms and one final.
+
+The course starts off extremely light, as the axioms of probability are often facts that students have known through prior experience. However, it will continually ramp up throughout the semester, with each topic building upon the previous one. Do not let this course blindside you - it can be extremely dangerous for your grade.
+
+During the semester, the course assigns 13 written homeworks, due every week. These homework sets are generally not too difficult and will contain calculations and fun scenarios. Students have cited that each week's homework can take very different amounts of time, from under an hour to upwards of five hours to complete. PrairieLearn homeworks are on the easier side - these used to be assigned as weekly 25-minute quizzes with two attempts for each problem, so as homeworks these should be significantly easier and very manageable.
+
+The course will have two midterms and a final, which each exam ramping up significantly from the previous one. There are usually a large bank of past practice exams. However, probability can be notoriously tricky - the questions on the actual exam can look nothing like past practice exams. Students are allowed to bring one double-sided 8.5x11 cheatsheet to the first exam, two double-sided 8.5x11 cheatsheets to the second exam, and so on. Students are highly recommended to reuse cheatsheets from prior midterms, but this is not enforced.
```

Comment:
the course -> ECE 313

(or replace one of the preceding three uses of "the/this course" with ECE313)

Reply:
use a colon after tricky